src/client/lib/_mapbox/map-config.js

```diff
@@ -0,0 +1,13 @@
+export const MAP_CENTER = [ 5.10, 52.09 ]
```

Comment:
add a comment `// center of NL`
or in code?:
```js
const NETHERLANDS_CENTER_LATITUDE = 5.10
const NETHERLANDS_CENTER_LONGITUDE = 52.09
export const MAP_CENTER = [
  NETHERLANDS_CENTER_LATITUDE,
  NETHERLANDS_CENTER_LONGITUDE,
]
```

Reply:
nice!